templates/init_sysv.erb

```diff
@@ -114,12 +114,13 @@ function stop() {
     if [ -f "${LOCK_DIR}/${NAME}" ]; then
       if [ "$RETVAL" -eq "0" ]; then
          touch ${LOCK_DIR}/${NAME} 2>&1 || RETVAL="4"
+         # we should read ${CATALINA_PID} here, because ${TOMCAT_SCRIPT} may overwrite ${CATALINA_PID}
+         read kpid < ${CATALINA_PID}
```

Comment:
We probably still need to check to make sure the pid file is there as we were doing below.